src/vs/platform/windows/electron-main/window.ts

```diff
@@ -286,6 +289,22 @@ export class CodeWindow extends Disposable implements ICodeWindow {
 				this._win.setSheetOffset(22); // offset dialogs by the height of the custom title bar if we have any
 			}
 
+			// Windows Custom System Context Menu
+			// See https://github.com/electron/electron/issues/24893
+			if (isWindows && useCustomTitleStyle) {
+				const WM_INITMENU = 0x0116;
+				this._win.hookWindowMessage(WM_INITMENU, () => {
+					const [x, y] = this._win.getPosition();
+					const cursorPos = screen.getCursorScreenPoint();
+
+					this._win.setEnabled(false);
+					this._win.setEnabled(true);
```

Comment:
These end up calling `EnableWindow` on windows with the toggled state https://docs.microsoft.com/en-us/windows/win32/api/winuser/nf-winuser-enablewindow. What is the reasoning to disable the window ?